Doc/using/cmdline.rst

```diff
@@ -669,6 +669,13 @@ Miscellaneous options
 
      .. versionadded:: 3.14
 
+   * :samp:`-X tlbc={0,1}` enables (1, the default) or disables (0) thread-local
+     bytecode in builds configured with :option:`--disable-gil`.  When disabled,
+     this also disables specialization in the tier 1 interpreter.  See also
```

Comment:
I think we stopped using the term "tier 1" and "tier 2" because it's too confusing with the tiers of buildbots. The new terms are just "specializing interpreter" and "JIT" respectively.

So maybe just call it "specializing interpreter"